common/dist/scripts/habitrpg-shared.js

```diff
@@ -2772,12 +2776,16 @@ api.spells = {
       target: 'task',
       notes: t('spellRogueBackStabNotes'),
       cast: function(user, target) {
-        var bonus, _crit;
+        var bonus, req, _crit;
         _crit = user.fns.crit('str', .3);
         target.value += _crit * .03;
         bonus = (target.value < 0 ? 1 : target.value + 1) * _crit;
         user.stats.exp += bonus;
-        return user.stats.gp += bonus;
+        user.stats.gp += bonus;
+        req = {
+          language: user.preferences.language
+        };
+        return user.fns.updateStats(user.stats, req);
```

Comment:
All the changes above are legit. They're from a previous recent change in which we mustn't have committed this compiled file.

FYI, it's on my task list for today to do a PR that changes .gitignore to exclude this file and then I'll leave it open for a while to see if anybody complains. :)